### ISSUE TITLE
Человек не должен брать два пистолета

### ISSUE THREAD

**Issue by mentoster** [RTUITLab/RevolIT#36]

**Comment by CAPCHIK** [comment]
А может просто спавнить больше пистолетов? С точки зрения веселья механику из двух пистолетов выглядит вполне живой

**Comment by mentoster** [comment]
> А может просто спавнить больше пистолетов? С точки зрения веселья механику из двух пистолетов выглядит вполне живой

Да, человек может и быть с двумя пистолетами, только нужно немного поменять механику, больше ставить пистолетов, добавить суммирование очков по двум пистолетам

**Comment by mentoster** [comment]
Тогда issue меняется на добавить больше пистолетов, исправить механику с пистолетами

**Comment by CAPCHIK** [comment]
Вот мне видится это более верным и красивым решением, нежели просто "нельзя брать второй пистолет, а та та", отлично
